src/Core/src/ImageSources/StreamImageSourceService/StreamImageSourceService.Android.cs

```diff
@@ -21,7 +21,7 @@ public partial class StreamImageSourceService
 				Stream? stream = null;
 				try
 				{
-					stream = await streamImageSource.GetStreamAsync(cancellationToken).ConfigureAwait(false);
+					stream = await streamImageSource.GetStreamAsync(cancellationToken);
```

Comment:
So the `ImageSetFromStreamRenders` test above fails unless we have this change?

Reply:
yes